compose/material3/material3/src/androidMain/kotlin/androidx/compose/material3/ExposedDropdownMenu.kt

```diff
@@ -289,7 +252,7 @@ interface ExposedDropdownMenuBoxScope {
  * Contains default values used by Exposed Dropdown Menu.
  */
 @ExperimentalMaterial3Api
-object ExposedDropdownMenuDefaults {
+actual object ExposedDropdownMenuDefaults {
```

Comment:
Instead of making it expect/actual, we can just fully copy it to commonMain. It will simplify uinderstanding and future merges

Reply:
Done (removed `Deprecated/HIDDEN` that were there for binary compatibility)